server/SocketManager/socketmanager.js

```diff
@@ -1,11 +1,14 @@
 let sockets = {};
+let socket_user_map = {};
 
-const updateSocket = (usertoken, clientsocket) => {
+const updateSocket = (usertoken, clientsocket, user_id) => {
   sockets[usertoken] = clientsocket;
+  socket_user_map[usertoken] = user_id;
 };
 
 const removeSocket = usertoken => {
   delete sockets[usertoken];
```

Comment:
The variable name `sockets` could be more descriptive. Consider renaming it to `activeSockets` to better convey its purpose.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*